.github/workflows/e2e-nvidia-a10g-x1.yml

```diff
@@ -156,7 +156,7 @@ jobs:
 
           python3.11 -m pip show nvidia-nccl-cu12
           
-          ./scripts/basic-workflow-tests.sh -mf
+          ./scripts/basic-workflow-tests.sh -mL
```

Comment:
Is `e2e-nvidia-a10g-x4` using this flag too?

Reply:
Checking right now

Reply:
Doesn't look like it